src/transformers/models/mamba/modeling_mamba.py

```diff
@@ -545,7 +545,6 @@ def forward(
         use_cache: Optional[bool] = None,
         output_hidden_states: Optional[bool] = None,
         return_dict: Optional[bool] = None,
-        **kwargs,  # `attention_mask` is passed by the tokenizer and we don't want it
```

Comment:
alternatively, we can accept `attention_mask` and raise an exception when it is not `None`